### DISCUSSION TITLE
Question: Build for productions module export issue

### DISCUSSION THREAD

**Discussion by muradsofi** [webpack/webpack#14133] [General]
So in my current project when I build with dev mode everything works fine but when I change the mode to production I started getting errors.  (We use modulefederation)

Error is when I use some module from other project it's return object instead of function. But object has default property which I can access my function. 

Why this happened and how we can solve it? 

Webpack version 

        "webpack": "^5.46.0",
        "webpack-cli": "^4.7.2",
        "webpack-dev-server": "^3.11.2",

**Comment by alexander-akait** [comment]
If you need a help please provide more information

**Reply by muradsofi** [reply]
What information do U need?

**Reply by muradsofi** [reply]
So interesting part of that I can use normal export imports but the problem happened with when I export/import from another module (another project) it's return object instead of function.

**Reply by alexander-akait** [reply]
Please provide steps to reproduce
